components/o-comments/src/js/stream.js

```diff
@@ -326,7 +328,22 @@ class Stream {
 					View our full ${this.options.linkSubscribe ? `<a class="linkSubscribe" href='${this.options.linkSubscribe}'>subscription packages</a>` : `subscription packages` } to join the conversation.
 				</p>
 			`;
-			customMessageContainer.innerHTML = this.isTrialist ? messageForTrial : this.isRegistered ? messageRegistered : messageForAnonymous;
+			const messageForEmailInUse = `
+			<h3>There is a problem configuring your commenting account</h3>
+				<p>
+					Please contact ${this.options.linkContact ? `<a class="linkContact" href='${this.options.linkContact}'>Customer Care</a>` : `Customer Care`} referencing issue code E403.
+				</p>
+			`;
+			if (this.isRegistered) {
+				customMessageContainer.innerHTML = messageRegistered;
+			} else if (this.isTrialist) {
+				customMessageContainer.innerHTML = messageForTrial;
+			} else if (this.emailInUse) {
+				customMessageContainer.innerHTML = messageForEmailInUse;
```

Comment:
As per my previous comment, this could code could be structured differently...

```suggestion
				customMessageContainer.innerHTML = `<h3>There is a problem configuring your commenting account</h3>
				<p>
					Please contact ${this.options.linkContact ? `<a class="linkContact" href='${this.options.linkContact}'>Customer Care</a>` : `Customer Care`} referencing issue code E403.
				</p>
			`;
```